code/modules/clothing/suits/armor.dm

```diff
@@ -476,6 +476,24 @@
 	siemens_coefficient = 0
 	price_tag = 600
 	//Used ablative gear armor values and technomancer helmet/voidsuit values.
+//onestar
+/obj/item/clothing/suit/storage/onestar/jacket
+	name = "OneStar officer jacket"
+	desc = "A rare stylish red jacket worn by onestar officals. it seems to be extremly durable and is strangely warm to the touch"
```

Comment:
```suggestion
	desc = "A rare stylish red jacket worn by One Star offciers. It seems to be extremly durable and is strangely warm to the touch."
```

Reply:
honestly that looks more like a coat to me

Reply:
Did the changes